src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -222,13 +222,12 @@ function MultiCombobox<Item>({
             onMouseDown: toggleMenu,
             readOnly: true,
             placeholder: !value.length ? placeholder : '',
-            position: !value.length ? 'static' : 'absolute',
           }),
           ...(searchable && {
-            placeholder,
-            mt: (isOpen && value.length) || (isOpen && value.length && allowAdditions) ? -4 : 0,
-            position: isOpen || (isOpen && value.length && allowAdditions) ? 'static' : 'absolute',
+            placeholder: hideLabel && value.length && !isOpen ? '' : placeholder,
+            p: isOpen ? 1 : null,
```

Comment:
why are we adding a padding here? Is it so we "align" the input to the blue tags?

Reply:
Correct.